apps/portals/elportal/src/config/routesConfig.tsx

```diff
@@ -127,6 +127,78 @@ const routes: RouteObject[] = [
           </SectionLayout>
         ),
       },
+      {
+        path: 'Comparative Biology',
+        element: (
+          <SectionLayout title={'Comparative Biology'}>
+            <Markdown
+              ownerId="syn27229419"
+              wikiId="630611"
+              loadingSkeletonRowCount={15}
+            />
+          </SectionLayout>
+        ),
+      },
+      {
+        path: 'Omnics Profiles in Humans',
+        element: (
+          <SectionLayout title={'Omnics Profiles in Humans'}>
+            <Markdown
+              ownerId="syn27229419"
+              wikiId="630616"
+              loadingSkeletonRowCount={15}
+            />
+          </SectionLayout>
+        ),
+      },
+      {
+        path: 'Translational Approaches',
+        element: (
+          <SectionLayout title={'Translational Approaches'}>
+            <Markdown
+              ownerId="syn27229419"
+              wikiId="630612"
+              loadingSkeletonRowCount={15}
+            />
+          </SectionLayout>
+        ),
+      },
+      {
+        path: 'AI/ML Models',
```

Comment:
Probably should not have a `/` in the path, could make this just `AI Models` and leave the title